requirements.yml

```diff
@@ -47,7 +47,7 @@
 
 - name: load-balancer
   src: https://github.com/open-craft/ansible-load-balancer
-  version: b729113c24d877aae44246ed4888eb73f773bd19
+  version: 52ae02307d18b7d693113d35523cb4ed045d3ff0
```

Comment:
I know the convention is to use a specific hash, but this will get difficult to manage pretty easily..  We already forgot to update it when https://github.com/open-craft/ansible-load-balancer/pull/7 merged, and now there's [2 more PRs open](https://github.com/open-craft/ansible-load-balancer/pulls).

Would it be ok to change this to `version: master`?